gobblin-service/src/test/java/org/apache/gobblin/service/GobblinServiceManagerTest.java

```diff
@@ -346,13 +346,14 @@ public void testRunQuotaExceeds() throws Exception {
 
   @Test (dependsOnMethods = "testRunQuotaExceeds")
   public void testExplainJob() throws Exception {
+    int sizeBeforeTest = this.gobblinServiceManager.getFlowCatalog().getSpecs().size();
     FlowConfig flowConfig = new FlowConfig().setId(new FlowId().setFlowGroup(TEST_GROUP_NAME).setFlowName(TEST_FLOW_NAME))
         .setTemplateUris(TEST_TEMPLATE_URI).setProperties(new StringMap(flowProperties)).setExplain(true);
 
     this.flowConfigClient.createFlowConfig(flowConfig);
 
     // explain job should not be persisted
-    Assert.assertEquals(this.gobblinServiceManager.getFlowCatalog().getSpecs().size(), 0);
+    Assert.assertEquals(this.gobblinServiceManager.getFlowCatalog().getSpecs().size(), sizeBeforeTest);
```

Comment:
does this presume a single-threaded test runner?  if so, was that already configured prior to this PR?